database/migrations/sqlite/20241209010200_add_lock_columns.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE hm_user_session 
+ADD COLUMN hm_version INT DEFAULT 1;
+
+ALTER TABLE hm_user_session 
+ADD COLUMN lock INT DEFAULT 0;
```

Comment:
Why lock is only in sqlite migration? I think the migrations need to be the same for all drivers/engines.

Reply:
Ok, I see, sqlite uses artificial locking via a column. Ignore this msg.